Reply/app/src/main/res/values/dimens.xml

```diff
@@ -28,6 +28,9 @@
     <dimen name="navigation_drawer_profile_image_size_padded">32dp</dimen>
     <dimen name="navigation_drawer_menu_item_height">56dp</dimen>
 
+    <!--HomeFragment-->
+    <dimen name="compose_email_fab_size">96dp</dimen>
+
```

Comment:
This can be removed since it comes for free with the new `large` fab style theme attrs

Reply:
https://github.com/material-components/material-components-android/blob/master/docs/components/FloatingActionButton.md#large-fabs